packages/itwinui-css/src/tile/tile.scss

```diff
@@ -73,6 +73,12 @@
         transform: translateZ(0) scale(1.1);
       }
     }
+
+    position: relative;
+
+    // .iui-tile-content {
+    //   position: unset;
+    // }
```

Comment:
Don't forget to remove this before merging.

Reply:
Removed :)